README.md

```diff
@@ -7,8 +7,10 @@ branch. If you are using SDK 2.x, we highly recommend planning your migration to
 Get started with Voice on Android:
 
 - [Quickstart](#quickstart) - Run the quickstart app
+- [New Features](Docs/new-features-4.0.md) - New features in 4.0
```

Comment:
Should we specifically say `4.0` in the bullet title, to distinguish from the the old `New Features`?

Reply:
yes @bchen-twilio .. updating